controllers/network.go

```diff
@@ -490,6 +490,12 @@ func createNetwork(w http.ResponseWriter, r *http.Request) {
 			// make host remote access gateway
 			logic.CreateIngressGateway(network.NetID, newNode.ID.String(), models.IngressRequest{})
 		}
+		// send peer updates
+		if servercfg.IsMessageQueueBackend() {
+			if err = mq.PublishPeerUpdate(false); err != nil {
```

Comment:
you can remove the `servercfg.IsMessageQueueBackend()` check

Reply:
done, please review again. Thanks.